src/ovp_pipeline/commands/init_user_context.py

```
@@ -0,0 +1,83 @@
+import sys
+from importlib.resources import files
+from pathlib import Path
+
+from ..context_loader import RULES_REL, USER_PROFILE_REL
+
+
+def _template_path(name: str) -> Path:
+    return Path(str(files("ovp_pipeline") / "data" / name))
+
+
+def _copy_template(template_name: str, target: Path, force: bool) -> str:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)